ibm/service/power/resource_ibm_pi_volume_clone.go

```diff
@@ -197,39 +217,17 @@ func isWaitForIBMPIVolumeCloneCompletion(ctx context.Context, client *st.IBMPICl
 	return stateConf.WaitForStateContext(ctx)
 }
 
-func isIBMPIVolumeCloneRefreshFunc(client *st.IBMPICloneVolumeClient, id string) resource.StateRefreshFunc {
+func isIBMPIVolumeCloneRefreshFunc(client *instance.IBMPICloneVolumeClient, id string) retry.StateRefreshFunc {
 	return func() (interface{}, string, error) {
 		volClone, err := client.Get(id)
 		if err != nil {
 			return nil, "", err
 		}
 
-		if *volClone.Status == VolumeCloneCompleted {
-			return volClone, VolumeCloneCompleted, nil
+		if *volClone.Status == State_Completed {
+			return volClone, State_Completed, nil
 		}
 
-		return volClone, VolumeCloneRunning, nil
-	}
-}
-
-func clonedVolumesSchema() *schema.Schema {
```

Comment:
there was no pattern to how it was written and which functions got it, and since the schema doesn't change very often I rather be able to see the whole thing at once in one file